config/dispatcher/dispatcher.ex

```diff
@@ -667,6 +667,11 @@ defmodule Dispatcher do
     Proxy.forward conn, path, "http://document-stamping/"
   end
 
+  ### PDF Signature Remover
+  match "/pdf-signature-remover/pieces/:piece_id/strip", @json_service do
```

Comment:
minor detail, `match` > `post`.
We don't do this often enough I think, it just prevents sending an unanswered `get`